projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -115,6 +120,43 @@ function is_wpcom() {
 	return defined( 'IS_WPCOM' ) && IS_WPCOM;
 }
 
+/**
+ * Adds a 'Newsletter' column after the 'Title' column
+ *
+ * @param array $columns An array of column names.
+ * @return array An array of column names.
+ */
+function add_newsletter_access_column( $columns ) {
+	$position   = array_search( 'title', array_keys( $columns ), true );
+	$new_column = array( NEWSLETTER_COLUMN_ID => '<span>' . __( 'Newsletter', 'jetpack' ) . '</span>' );
+	return array_merge( array_slice( $columns, 1, $position ), $new_column, array_slice( $columns, $position ) );
+}
+
+/**
+ * Displays the newsletter access level.
+ *
+ * @param string $column_id The ID of the column to display.
+ * @param int    $post_id The current post ID.
+ */
+function populate_newsletter_access_rows( $column_id, $post_id ) {
+	if ( NEWSLETTER_COLUMN_ID !== $column_id ) {
+		return;
+	}
+
+	$access_level = get_post_meta( $post_id, META_NAME_FOR_POST_LEVEL_ACCESS_SETTINGS, true );
+
+	switch ( $access_level ) {
+		case Token_Subscription_Service::POST_ACCESS_LEVEL_PAID_SUBSCRIBERS:
+			echo esc_html__( 'Paid Subscribers', 'jetpack' );
+			break;
+		case Token_Subscription_Service::POST_ACCESS_LEVEL_SUBSCRIBERS:
+			echo esc_html__( 'Subscribers', 'jetpack' );
+			break;
+		default:
```

Comment:
This does not align with https://github.com/Automattic/wp-calypso/pull/76382/files.
We miss the "Everybody" case. My previous comment was about missing the "default" case, when no access level was defined.

Then it should be good to go!

Reply:
Thanks, fixed in [ad75cd9](https://github.com/Automattic/jetpack/pull/30172/commits/ad75cd9935896c11a11a82455b20ae3786315367)